### PR TITLE
Updated jobit date and CV forms

### LINKED ISSUE #83: 2021 / Change dates to 2021 edition

### LINKED ISSUE #91: Create and add a form for CV submission

### PR DESCRIPTION
**Pull request by FilipePintoReis**
Closes #91 
Closes #83

### CODE REVIEW

**Comment by ZeCarlosCoutinho** [comment]
Not directly code related, but the form shows both English and Portuguese:
image
Is this something automatic from Google or was it a "typo"?

File: _config.yml [hunk 1]
```diff
@@ -44,25 +44,25 @@ organizerLink: "http://alumniei.pt/"
 navigationLinks:
  #- {permalink: "/", text: "Home"}
 bottomNavigationLinks:
- - {link: "https://forms.gle/as4vi5bJuai7XZnr5", text: "Share your CV"}
+ - {link: "https://forms.gle/SJTv7bZHUJNUULd57", text: "Share your CV"}
```
Comment (ZeCarlosCoutinho) [inline _config.yml:47]: Couldn't you make the form url a variable?
See [this](https://stackoverflow.com/questions/41620674/use-placeholders-in-yaml)

File: _config.yml [hunk 2]
```diff
@@ -44,25 +44,25 @@ organizerLink: "http://alumniei.pt/"
...
 rightNavigationButtons:
- - {link: "https://forms.gle/as4vi5bJuai7XZnr5", text: "Share your CV"}
+ - {link: "https://forms.gle/SJTv7bZHUJNUULd57", text: "Share your CV"}
```
Comment (ZeCarlosCoutinho) [inline _config.yml:49]: same

File: _config.yml [hunk 3]
```diff
@@ -44,25 +44,25 @@ organizerLink: "http://alumniei.pt/"
...
 heroButtons:
 #- {link: "https://bit.ly/dfua-c4p", text: "Send your CV"}
 # - {permalink: "/#tickets", text: "Buy tickets"}
 
 # About Block
-aboutTitle: "25th, 27th and 29th of May"
+aboutTitle: "5th, 12th and 19th of May"
 aboutBlock:
- - {title: "Who knows? Maybe your career starts here!", link: "https://forms.gle/as4vi5bJuai7XZnr5", buttonText: "Share your CV", text: "Your CV will be sent to the companies participating in the event. " }
+ - {title: "Who knows? Maybe your career starts here!", link: "https://forms.gle/SJTv7bZHUJNUULd57", buttonText: "Share your CV", text: "Your CV will be sent to the companies participating in the event. " }
```
Comment (ZeCarlosCoutinho) [inline _config.yml:65]: same

File: _config.yml (hunk 1 above)
Comment (ZeCarlosCoutinho) [reply _config.yml:47]: I mean, declare the variable at the beginning of the `_config.yml` and then use the variable where needed.

**Comment by FilipePintoReis** [comment]
> Not directly code related, but the form shows both English and Portuguese:
> image
> Is this something automatic from Google or was it a "typo"?

This is automatic, I'm afraid I can't do anything about it.

**Comment by FilipePintoReis** [comment]
Fixed

**Pull request merged**